cpufreq/src/cpufreq-applet.c

```diff
@@ -873,6 +887,14 @@ cpufreq_applet_refresh (CPUFreqApplet *applet)
 	horizontal = (applet->orient == MATE_PANEL_APPLET_ORIENT_UP ||
 		      applet->orient == MATE_PANEL_APPLET_ORIENT_DOWN);
 
+	/* We might be here for any number of reasons, including a font change,
```

Comment:
Increasing font size actually still truncates the label... you'll need https://github.com/mate-desktop/mate-applets/pull/274 for that.